src/containers/Tenant/Diagnostics/Partitions/Partitions.scss

```diff
@@ -5,11 +5,12 @@
     flex-grow: 1;
 
     height: 100%;
+    margin-right: -20px;
```

Comment:
`padding-right: 0px` to `&__table` should do the same

Reply:
it seems we don't need it at all